apollo-api/src/commonMain/kotlin/com/apollographql/apollo3/api/http/Http.kt

```diff
@@ -39,7 +40,8 @@ private constructor(
     val body: HttpBody?,
 ) {
 
-  fun newBuilder() = Builder(
+  @JvmOverloads
+  fun newBuilder(url: String = this.url, method: HttpMethod = this.method) = Builder(
```

Comment:
(micro nitpick: they're not in the same order, in the Builder, you have `method` first)

```suggestion
  fun newBuilder(method: HttpMethod = this.method, url: String = this.url) = Builder(
```

Reply:
Good catch, thanks!